cvat/apps/dataset_manager/formats/utils.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os.path as osp
+
+from datumaro.cli.util import make_file_name
+from datumaro.util.mask_tools import generate_colormap
+
+
+DEFAULT_COLORMAP_CAPACITY = 2000
+DEFAULT_COLORMAP_PATH = osp.join(osp.dirname(__file__), 'predefined_colors.txt')
+def parse_default_colors(file_path=None):
+    if file_path is None:
+        file_path = DEFAULT_COLORMAP_PATH
+
+    colors = {}
+    with open(file_path) as f:
+        for line in f:
+            line = line.strip()
+            if not line or line[0] == '#':
+                continue
+            _, label, color = line.split(':')
+            colors[label] = tuple(map(int, color.split(',')))
+    return colors
+
+def normalize_label(label):
```

Comment:
The function is not used anymore, but it should be to avoid problems with symbol case and special characters.

The logic is:
- sort labels to provide stable results (across different tasks with same labels)
- find matching labels using normalization
- if have conflicts, the first match uses the table
- generate random colors for the rest

Reply:
Now colors are set at  the time of label creation. So i guess sorting by its name can not provide stable result now. I can not create labels with the same name, so are there any conflicts?

Reply:
Conflicts appear from labels like `BACKGROUND`, `_BA_CK_gro_und_`, `back_GROUND` etc after their normalization. I feel that sorting might be unnecessary anymore, or should be done in label creation code, when it is done in batch mode.